### PR TITLE
Add api call to show track chooser

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

> What was changed in this pull request?

This PR adds an API call to let an external program show the track chooser (yellow overlay where the mouse is). It also enables the track chooser to respond to drag events in addition to mouse move events.

> Why is it necessary?

If we want to be able to drag something over a track we need a way to be able to show which track we're currently over.

## Screen capture

Sep-07-2019 14-49-22

Fixes #___

## Checklist

- [x] Set proper GitHub labels (e.g. v1.6+, ignore if you don't know)
- [ ] Unit tests added or updated
- [x] Documentation added or updated
- [x] Example(s) added or updated
- [ ] Update schema.json if there are changes to the viewconf JSON structure format
- [x] Screenshot for visual changes (e.g. new tracks or UI changes)
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: app/scripts/TiledPlot.js [hunk 1]
```diff
@@ -495,19 +495,18 @@ class TiledPlot extends React.Component {
     });
   }
 
-  handleOverlayMouseLeave() {
-    this.setState({
-      mouseOverOverlayUid: null,
-    });
+  handleOverlayMouseLeave(uid) {
+    if (uid === this.state.mouseOverOverlayUid) {
```
Comment (flekschas) [inline app/scripts/TiledPlot.js]: Why is this needed? Shouldn't it not matter which track is left?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
+## Next release
+
+- Add API call for show track chooser
```
Comment (flekschas) [inline CHANGELOG.md]: Could you add a pointer what this API is called? Maybe something like `hgApi. showTrackChooser()`?

File: app/scripts/TiledPlot.js (hunk 1 above)
Comment (pkerpedjiev) [reply app/scripts/TiledPlot.js]: It's necessary because of the order of events. Sometimes the dragStart event begins on an element before mouseLeave is called on the leaving element and that prevents the overlay from appearing. Took a surprisingly long time to figure that out actually.

File: CHANGELOG.md (hunk 2 above)
Comment (pkerpedjiev) [reply CHANGELOG.md]: Yup 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
```
Comment (pkerpedjiev) [inline CHANGELOG.md]:
```suggestion
- Add API call for show track chooser (hgApi. showTrackChooser()`)
```
